code/frameworks/angular/src/server/angular-cli-webpack.js

```diff
@@ -85,6 +85,10 @@ exports.getWebpackConfig = async (baseConfig, { builderOptions, builderContext }
     ]
   );
 
+  if (!builderOptions.experimentalZoneless && !cliConfig.entry.polyfills.includes('zone.js')) {
```

Comment:
Is `polyfills` always part of the `cliConfig`?  

What if angular libraries don't define it (because they can't). Should it then be this?

```typescript
!cliConfig.entry.polyfills?.includes('zone.js')
```